codemods/jscodeshift/immutable/4/isterable-to-iscollection/index.js

```diff
@@ -0,0 +1,42 @@
+/*
+The MIT License (MIT)
+
+Copyright (c) 2023 Rajasegar Chandran
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+
+/*
+Changes to the original file: added options
+*/
+
+
+module.exports = function transformer(file, api, options) {
+	const j = api.jscodeshift;
+
+	const root = j(file.source);
+
+	root.find(j.Identifier)
+    .filter(path => path.node.name === 'isIterable')
+      .replaceWith(j.identifier('isCollection'));
+
+	return root.toSource(options);
```

Comment:
check the code in this repo for `dirtyFlag`, you can check if the replacing was actually done and only then call `root.toSource()`

Reply:
resolved in #25 in [changed to ts, fixed comments](https://github.com/intuita-inc/codemod-registry/pull/25/commits/5e1f4bac2967053c351570cb8ff711c367c49161)